spec/graphql/mutations/trip_items/trip_item_delete_spec.rb

```diff
@@ -3,6 +3,8 @@
 RSpec.describe 'TripItemDelete', type: :request do
   describe '.resolve' do
     before :each do
+      #let let!(create variable before its called)
+      #black box vs clear box testing(industry - black box)
```

Comment:
Just a reminder that these comments are here! Would be a great opportunity to experiment with `let`